src/main/java/hudson/plugins/scm/koji/KojiChangeLogSet.java

```diff
@@ -83,4 +78,67 @@ public User getAuthor() {
 
     }
 
+    public static class HyperlinkStringContainer {
+
+        private List<HyperlinkString> hyperlinks;
+
+        public HyperlinkStringContainer(String string) {
+            hyperlinks = new ArrayList();
```

Comment:
You get compiler warnings with this. Safer to use `new ArrayList<>()`. IDE (at least IDEA) should notify about that.